### ISSUE TITLE
Description should include imports

### ISSUE THREAD

**Issue by kellijohnson** [nwfsc-assess/nwfscSurvey#7]
The [Description](https://github.com/nwfsc-assess/nwfscSurvey/blob/development/DESCRIPTION) file should include 
```
Imports: 
  chron,
  jsonlite
```
and other packages that are included in the `@import` call within the function documentation. This will facilitate the downloading of these packages automatically for people that are trying to install `nwfscSurvey`.

**Comment by kellijohnson** [comment]
Do we need `reshape2` as well? I am not sure where it is used in [`SurveyAgeAtLen.fn.R`](https://github.com/nwfsc-assess/nwfscSurvey/blob/development/R/SurveyAgeAtLen.fn.R).

**Comment by CWetzel** [comment]
Yes, the dcast function is within the reshape2 package.  I had completely forgotten about that package.  I have added this to the description file.  I did a quick scan of the rest of the files and this should be all the required packages.

**Comment by taylori** [comment]
@kellijohnson for future reference, how are you finding these issues, using `devtools::check()` or some other method?
